src/addons/dltx.md

```diff
@@ -65,7 +67,7 @@ You can only override sections that already have a base definition. Both loading
     >specific_characters_files                = character_desc_ah, character_desc_hidden_threat
   ```
 
-#### Converting already existing mods
+## Converting Already Existing Mods
```

Comment:
Actually, I like normal case (only first letter capitalized) more. The inconsistency comes from a lack of inspection, but if we fix it I'd like it to be the other way :)